lstchain/data/lstchain_standard_config.json

```diff
@@ -237,6 +237,7 @@
 
   "LSTCalibrationCalculator":{
     "systematic_correction_path": null,
+    "npe_median_cut_outliers": [-4,4],      
```

Comment:
where are these numbers coming from?
Also: `[-4, 4]`

Reply:
From statistical considerations, but you are right I am now working with a more conservative 5 sigma cut, i.e : a pixels is tagged as unusable if the estimated number of phe is 5 sigma far from the median camera value (keep in mind that de camera median dispersion is  7%).

Reply:
so shall they be changed to 5?

Reply:
Done